cmd/apiKeyValueMapUpload_test.go

```diff
@@ -0,0 +1,145 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRunApiKeyValueMapUpload(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Successfull Api Key Value Map Create Test", func(t *testing.T) {
+		// init
+		apiServiceKey := `{
```

Comment:
_apiServiceKey_ and _apiKeyValueMapUploadOptions_ are always the same, aren't they?

Proposal: refactor to _getDefaultOptions_().

Reply:
both are different objects. apiServiceKey is json text , where as apiKeyValueMapUploadOptions  is a structure which has more details, such as apiServiceKey, key, value, keyValueMapName.

Reply:
something like ?
```
getDefaultOptions {
  return  apiKeyValueMapUploadOptions{
			APIServiceKey:   {
			"oauth": {
				"url": "https://demo",
				"clientid": "demouser",
				"clientsecret": "******",
				"tokenurl": "https://demo/oauth/token"
			}
		}`,
			Key:             "demo",
			Value:           "name",
			KeyValueMapName: "demoMap",
		}
}
```

Reply:
done